backend/add_fish.py

```diff
@@ -0,0 +1,42 @@
+from app import app, fish_db, users_db, species_db
+from flask import make_response, request, redirect, url_for, abort
+from bson.objectid import ObjectId
+import flask_login as fl
+
+# login part
+logged_users = set()
+
+@fl.login_required
+@app.route("/add-fish", methods=["POST"])
```

Comment:
We should standarized the way how we name different entrypoints - for example we are not consistent with floor(_) and minus(-) in entrypoint naming.

Reply:
This is something we've already discussed with LogicalInsanity and have agreed upon dashes as route separators. This adheres to the existing standards.